### DISCUSSION TITLE
Categories in Search differ from labels/folders etc

### DISCUSSION THREAD

**Discussion by Hanskes** [photoprism/photoprism#1672] [Q&A] [answered]
When I try to search via Search (top left corner) I can filter by categories, it is not clear to me how I can assign these categories to photos, labels or folders. Categories assigned to labels or folders differ from the categories you can filter by using the search option and are not visible there. Where can I change and assign the categories that you can filter on in the search option?

**Comment by graciousgrey** [comment, marked as answer]
At the moment categories are groups of labels e.g. the labels peacock, chicken and ostrich belong to the category bird.
So if you assign one of those labels to a photo (https://docs.photoprism.org/user-guide/organize/labels/#add-labels-to-photos), it will get the category bird. 
Not all labels belong to a category and categories itself cannot be set or customized. 

You can add custom labels to your photos and use the label or keywords search filter. All labels are set as keywords automatically.
You can search for `label:cat`  or `keywords:cat` or even `keywords:"cat&dog"` or `keywords:"cat|dog"`

See: https://docs.photoprism.org/user-guide/organize/search/#and-search

**Comment by Hanskes** [comment]
Hi Graciousgrey, 
Thanks for your explanation. Is it possible that the possibility to create categories yourself and assign groups of labels to them will be added in the future? Would be very useful.

**Comment by lastzero** [comment]
Yes, providing a UI seems possible. Advanced users can add it manually in the database. It's just a connection between two labels, cat for example has the category animal. You can both see them in Labels (might have to click on the visibility toggle upper right). We didn't want to put all 1000 labels in the dropdown, just the major categories. It's equivalent to typing the label / category name in the regular search field.
